app/models/concerns/allinson_flex/dynamic_metadata_behavior.rb

```diff
@@ -91,7 +91,8 @@ def dynamic_schema=(value)
     def dynamic_schema_service(args = {})
       # clear memoizaiton when as_id changes
       old_as_id = @as_id
-      @as_id = args[:as_id] || admin_set_id || AdminSet::DEFAULT_ID
+      
+      @as_id = args[:as_id] || try(:admin_set_id) || AdminSet::DEFAULT_ID
```

Comment:
Is this change related to this admin_only change?

Reply:
removed